### DISCUSSION TITLE
How to read figure caption from DOCX

### DISCUSSION THREAD

**Discussion by rgaiacs** [jgm/pandoc#9390] [Q&A]
Pandoc has support to Microsoft Word's native table caption. For example, when [mwe-table.docx](https://github.com/jgm/pandoc/files/14099222/mwe-table.docx)

![Screenshot 2024-01-30 145920](https://github.com/jgm/pandoc/assets/1506457/155df23c-f137-403e-8cee-d7d7192a464d)

is converted using `pandoc --from docx --to html mwe-table.docx`, it produces

```
<p>Lorem ipsum.</p>
<table>
<caption><p>Awesome table</p></caption>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>A</th>
<th>B</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>2</td>
</tr>
</tbody>
</table>
```

**Does Pandoc has support to read Microsoft Word's native figure caption?** For example, when [mwe-img.docx](https://github.com/jgm/pandoc/files/14100099/mwe-img.docx)

![Screenshot 2024-01-30 155653](https://github.com/jgm/pandoc/assets/1506457/43d5143f-1521-4fb8-84d3-7d8844a32e04)

is converted using `pandoc --from docx --to html mwe-img.docx`, it produces

```
<p>Lorem ipsum.</p>
<p><img src="media/image1.png"
style="width:1.10263in;height:1.10263in" /></p>
<p>Figure Blue square.</p>
```

instead of

```
<p>Lorem ipsum.</p>
<figure>
   <img src="media/image1.png"/>
   <figcaption>
      Blue square.
   </figcaption>
</figure>
```

**Comment by jgm** [comment]
Agreed, let's make this an issue.

**Comment by jgm** [comment]
See #9391
